src/main/web_src/src/components/fagsystem/udistub/form/partials/IkkeOppholdSammeVilkaar.tsx

```diff
@@ -97,10 +97,26 @@ export const IkkeOppholdSammeVilkaar = ({ formikBag }: any) => (
 				}
 				label="Innreiseforbud"
 				options={Options('jaNeiUavklart')}
+				onChange={(event: any) => {
+					formikBag.setFieldValue(
+						'udistub.oppholdStatus.ikkeOppholdstilatelseIkkeVilkaarIkkeVisum.utvistMedInnreiseForbud.innreiseForbud',
+						event.value
+					)
+					if (event.lowercaseLabel.includes('nei') || event.lowercaseLabel.includes('uavklart')) {
```

Comment:
Må ha en sjekk for null også, siden verdien i feltet kan fjernes. 😁 
Og kanskje det hadde vært fint å flytte hele greia ut som en egen funksjon, så slipper vi så mye rot i formet? 🤔

Reply:
Haha sant! Jeg som trodde jeg hadde testa alt å greier 😂